core/src/test/java/org/opensearch/sql/data/model/DateTimeValueTest.java

```diff
@@ -136,41 +123,12 @@ public void timestampInUnsupportedFormat() {
         exception.getMessage());
   }
 
-  @Test
-  public void datetimeInUnsupportedFormat() {
-    SemanticCheckException exception =
-        assertThrows(
-            SemanticCheckException.class, () -> new ExprDatetimeValue("2020-07-07T01:01:01Z"));
-    assertEquals(
-        "datetime:2020-07-07T01:01:01Z in unsupported format, "
-            + "please use 'yyyy-MM-dd HH:mm:ss[.SSSSSSSSS]'",
-        exception.getMessage());
-  }
-
-  @Test
-  public void stringDateTimeValue() {
-    ExprValue stringValue = new ExprStringValue("2020-08-17 19:44:00");
-
-    assertEquals(LocalDateTime.parse("2020-08-17T19:44:00"), stringValue.datetimeValue());
```

Comment:
You can keep this test, but check timestamp value instead of datetime

Reply:
Good point, added the test back in and updated to work with timestamp